packages/frontend/app/models/message/utils.ts

```diff
@@ -11,7 +11,13 @@ export function selectUnreadDirectMessages(messages: Message[], fromId: string)
 export function selectUnreadMessages(messages: Message[]) {
   const filtered = messages.filter(m => {
     return (
-      m.unread && m.target !== TARGET.NONE && m.target !== TARGET.MESSAGE && m.type !== TYPE.PING
+      m.unread &&
+      m.target !== TARGET.NONE &&
+      m.target !== TARGET.MESSAGE &&
+      m.type !== TYPE.PING &&
+      // ember-data in-flight messages
+      // don't yet have any fields
+      m.from
```

Comment:
This `m.from` check should probably happen first, since it'll short circuit the rest of the condition